<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientPreferencesDialog</class>
 <widget class="QDialog" name="ClientPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FF6B6B;  /* Culoare primarÄƒ */
    color: white;
    border-radius: 15px;
    padding: 10px 20px;
    border: 2px solid #FF8E8E;
    font-weight: bold;
    min-width: 120px;
}

QPushButton:hover {
    background-color: #FF8E8E;  /* Culoare la hover */
}

QPushButton:pressed {
    background-color: #FF5252;  /* Culoare la click */
}</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="allergiesEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="foodPreferencesEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="deliveryTimeCombo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="notesEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pricePreferenceCombo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
